tortoise/functions.py

```diff
@@ -110,10 +113,32 @@ def resolve(self, model: "Type[Model]", table: Table) -> dict:
 class Aggregate(Function):
     """
     Base for SQL Aggregates.
+
+    :param field: Field name
+    :param default_values: Extra parameters to the function.
+    :param is_distinct: Flag for aggregate with distinction
     """
 
     database_func = AggregateFunction
 
+    def __init__(
+        self,
+        field: Union[str, F, ArithmeticExpression],
+        *default_values: Any,
+        is_distinct=False,
```

Comment:
@lntuition Now here is a valid use-case to keep `Aggregate` separate from `Function`.
A function like `Length`, `Trim`, etc... doesn't make sense with a `distinct`, but aggregates really do.

Reply:
@pjongy I feel an interface of `Count('somecol', distinct=True)` would read/write better.